src/app/static/src/app/store/plottingSelections/getters.ts

```diff
@@ -16,7 +16,8 @@ export const getters = {
                 return {}
         }
     },
-    calibratePlotDefaultSelections: (state: PlottingSelectionsState, getters: any, rootState: RootState): BarchartSelections => {
-        return rootState.modelCalibrate.calibratePlotResult!.plottingMetadata.barchart.defaults;
+    calibratePlotDefaultSelections: (state: PlottingSelectionsState, getters: any, rootState: DataExplorationState): BarchartSelections => {
+        // TODO find solution to this hack
```

Comment:
this is the only place where a module we need for both modes spans multiple steps - options would be to leave something like this in, I guess using the flag:

```
if (!rootState.dataExplorationMode) {
    return (rootState as RootState).modelCalibrate.calibratePlotResult!.plottingMetadata.barchart.defaults;
}
```

 or to split this module up, maybe move plotting selection getters into the modules where they apply (i.e. this would live in `modelCalibrate`, the above in `surveyAndProgram` etc)

Reply:
Yes, I wasn't proposing more modules, but moving these methods into existing modules. Probably removing this module completely is not worth the faff but this getter actually only references state from the `modelCalibrate` module, not from the `plottingSelections` module at all! So it would make sense to just move it to be a `modelCalibrate` getter. The casting is prettty hack-y because it involves contradicting the type declaration.

Reply:
Similarly, although not needed for this change, the first getter in this class only references state from the `surveyAndProgram` module, so should probably be a getter on that module instead.